packages/backend/package.json

```diff
@@ -17,23 +17,23 @@
   },
   "dependencies": {
     "app": "link:../app",
-    "@backstage/backend-common": "^0.18.5",
-    "@backstage/backend-tasks": "^0.5.2",
-    "@backstage/catalog-model": "^1.0.3",
-    "@backstage/catalog-client": "^1.0.3",
-    "@backstage/config": "^1.0.1",
-    "@backstage/plugin-app-backend": "^0.3.33",
-    "@backstage/plugin-auth-backend": "^0.18.3",
-    "@backstage/plugin-catalog-backend": "^1.2.0",
-    "@backstage/plugin-permission-common": "^0.7.5",
-    "@backstage/plugin-permission-node": "^0.7.8",
-    "@backstage/plugin-proxy-backend": "^0.2.27",
-    "@backstage/plugin-scaffolder-backend": "^1.3.0",
-    "@backstage/plugin-search-backend": "^1.3.1",
-    "@backstage/plugin-search-backend-module-pg": "^0.5.6",
-    "@backstage/plugin-search-backend-node": "^1.2.1",
-    "@backstage/plugin-techdocs-backend": "^1.1.2",
-    "@janus-idp/backstage-plugin-apicurio-backend":"link:../../plugins/apicurio-backend",
+    "@backstage/backend-common": "^0.19.0",
+    "@backstage/backend-tasks": "^0.5.3",
+    "@backstage/catalog-model": "^1.4.0",
+    "@backstage/catalog-client": "^1.4.2",
+    "@backstage/config": "^1.0.8",
+    "@backstage/plugin-app-backend": "^0.3.46",
+    "@backstage/plugin-auth-backend": "^0.18.4",
+    "@backstage/plugin-catalog-backend": "^1.10.0",
+    "@backstage/plugin-permission-common": "^0.7.6",
+    "@backstage/plugin-permission-node": "^0.7.9",
+    "@backstage/plugin-proxy-backend": "^0.2.40",
+    "@backstage/plugin-scaffolder-backend": "^1.15.0",
+    "@backstage/plugin-search-backend": "^1.3.2",
+    "@backstage/plugin-search-backend-module-pg": "^0.5.7",
+    "@backstage/plugin-search-backend-node": "^1.2.2",
+    "@backstage/plugin-techdocs-backend": "^1.6.3",
+    "@janus-idp/backstage-plugin-apicurio-backend": "link:../../plugins/apicurio-backend",
```

Comment:
Noticing this now ... Should we properly publish the dependency instead of using a local link?

Reply:
Yes, but the dependency will be under the janus-idp org...they are going to publish the dependency...so for the "showcase" we can go with the local plugin.

Other topic is the way that you are going to manage this:

- Create directly the PR in janus from your local fork

or

- Consolidate here the features and then make the PR from this repo.

Reply:
I'm ok with starting in this repo, but we probably should start to have the conversation to get a project in the `janus-idp` organization. cc. @EricWittmann